README.md

```diff
@@ -1,20 +1,21 @@
 # APP PhotoNavTex
-App for nautical warning messages
+
+At the end of neufische intensive coding bootcamp (https://www.neuefische.de/en/weiterbildung/java/hamburg) I have implemented this app for nautical warning messages.
+
+NAVTEX is  an international system for maritime safety information (MSI) which includes navigational warnings, weather forecasts and weather warnings, search and rescue notices and similar information to ships. PhotoNavTex helps to indicate two types of these warnings: **A** (navigational warnings) and **D** (Search And Rescue information and pirate attack warnings). Therefore it displays the position of saved warnings on the OpenStreetMap as basis map and on OpenSeaMap as overlay.
```

Comment:
I would say 
"[...] similar information for ships."
and "PhotoNavTex helps to indicate two types of warnings [...]"
and "It displays the position of saved warnings on the [...]"